### ISSUE TITLE
Searchable selects: search in the middle of the strings

### ISSUE THREAD

**Issue by lucasromanojf** [livewire/flux#85]
Hello,

currently the searchable select searches only in the beginning of the strings. It would be great if it could search in the middle of them.

**Comment by wevertondcl** [comment]
Yes, I agree! Additionally, some words in Portuguese (Pt-BR) have accents, like names such as João and Márcia. If you don't type the accent, the search won't return results

**Comment by lucasromanojf** [comment]
Same here, I am from Brazil too 😁

**Comment by lucasromanojf** [comment]
> Yes, I agree! Additionally, some words in Portuguese (Pt-BR) have accents, like names such as João and Márcia. If you don't type the accent, the search won't return results


Same here, I am from Brazil too 😁

**Comment by LasseRafn** [comment]
Relates to #14 btw
